dmorg.tex

```diff
@@ -150,6 +150,32 @@ \subsubsection{Hardware Configuration Control}
 
 The sizing model for compute hardware purchasing is detailed in \citeds{LDM-144}, \citeds{LDM-141}, and \citeds{LDM-138}.
 
+\subsection {Release Management } \label{sect:release}
+
+
+The DMCCB maintains  the release plan, \citeds{LDM-564},  synchronized with  the project milestones.
+
+All releases will be identified by a release issue - but we wish to move to Jira release functionality.
+
+Any unscheduled  release, major, minor or patch, needs to be requested by the end user to the DMCCB using a RFC Jira issue. The RFC shall contain:
```

Comment:
Where is the schedule for releases published? In particular, I can think of two schedules at the moment (releases in P6 and milestones in LDM-564) which don't agree in detail.

Reply:
I think that P6 should be the source of true.
LDM-564 shall just provide access to this information.